scripts/prepare_ml_data.py

```diff
@@ -44,50 +44,82 @@
 
 logging.getLogger("nowcasting_dataset.data_source").setLevel(logging.WARNING)
 
-# load configuration, this can be changed to a different filename as needed
-filename = os.path.join(os.path.dirname(nowcasting_dataset.__file__), "config", "gcp.yaml")
+ENABLE_NEPTUNE_LOGGING = False
+
+# load configuration, this can be changed to a different filename as needed.
+# TODO: Pass this in as a command-line argument.
```

Comment:
when this is done can we use [Click](https://click.palletsprojects.com/en/8.0.x/). Is it worth making an issue for this? and then putting that issue number int eh code. Keeps the TODOs a bit tidier

Reply:
I second Click!

Reply:
Good idea!  And thanks for letting me know about Click - I hadn't seen that before!

I've started an issue [here](https://github.com/openclimatefix/nowcasting_dataset/issues/171).

I'll add a link in the code to the issue.